Dockerfile

```diff
@@ -35,7 +35,7 @@ ARG PKGS_LIST=main-packages-list.txt
 
 COPY ${PKGS_LIST} /tmp/main-packages-list.txt
 
-RUN dnf install -y python3 python3-requests && \
+RUN dnf install -y epel-release python3 python3-requests && \
```

Comment:
EPEL may conflict with TripleO repos, I'd rather avoid having it. It will be no-go for Red Hat downstream as well.

Reply:
Oh, well.. Back to square 1 then.